debian/control

```diff
@@ -9,7 +9,7 @@ Homepage: https://jitsi.org
 
 Package: jigasi
 Architecture: any
-Depends: ${misc:Depends}, openjdk-11-jre-headless | openjdk-11-jre | java11-runtime-headless | java11-runtime, libxss1, openssl, ruby-hocon
+Depends: ${misc:Depends}, default-jre-headless (>= 2:1.11), openssl, ruby-hocon
```

Comment:
What's with the ruby-hocon thing?

Reply:
https://github.com/jitsi/jigasi/blob/296b7cd94c0712d586737b1e59dd8d12f453a475/debian/postinst#L163

Reply:
And of course the:
https://github.com/jitsi/jigasi/blob/296b7cd94c0712d586737b1e59dd8d12f453a475/debian/postinst#L164

Reply:
Urgh.